packages/webapp/src/containers/Task/constants.js

```diff
@@ -25,3 +25,7 @@ export const TASK_TYPES = {
   HARVEST: 'harvest_tasks',
   IRRIGATION: 'irrigation_task',
 };
+
+export const TASKTYPE_PRODUCT_MAP = {
```

Comment:
Optional improvement to make this a function. All future task products should be `${taskType}_products`.